src/components/Lend/AaveAction.tsx

```diff
@@ -0,0 +1,46 @@
+import { TxBuilderV2, Network, Market } from "@aave/protocol-js";
+import { ethers } from "ethers";
+
+export const deposit = async (provider, accountAddress) => {
+  let txBuilder;
```

Comment:
Nit: You can move this down and just initialize/declare in the same line

Reply:
cleaned up